### PR TITLE
move continuous integration tests to GitHub Actions

### PR DESCRIPTION
**Pull request by willnorris**
This allows bringing both linux and windows CI tests into a single platform, and sharing accounts/permissions with the rest of the repo rather than separated out on Travis and AppVeyor.  If we also use GitHub Actions for future automations like release scripts, updating the website, etc, then having CI here as well means only a single system to learn.

Performance is about the same as Travis and AppVeyor... about 5-6 minutes for linux tests to complete, about 8 minute for Windows.  The failing Windows tests here are the same ones that were already failing.  I have a set of changes queued up that will address all of those.

### CODE REVIEW

**Comment by willnorris** [comment]
Example test results can be seen at https://github.com/willnorris/perkeep/actions

**Comment by bradfitz** [comment]
Hell yeah, thanks!

File: .github/workflows/tests-linux.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+on: [push, pull_request]
+name: "tests/linux"
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.x, 1.13.x]
```
Comment (bradfitz) [inline .github/workflows/tests-linux.yml:8]: Don't we require Go 1.14 at least?

**Review by bradfitz — APPROVED** [review]

File: .github/workflows/tests-linux.yml (hunk 1 above)
Comment (willnorris) [reply .github/workflows/tests-linux.yml:8]: still says 13 here: https://github.com/perkeep/perkeep/blob/7321b90b6e112e7847cb6f3f9683c74d545d5059/make.go#L901

**Review by mpl — COMMENTED** [review]
Is there zero advantage in keeping the Travis runs around as well? Say for when github craps itself for example?

**Comment by willnorris** [comment]
I don't really think we need to keep it around.  And if GitHub really does fall down in some terrible way, we can always bring back the travis config.

**Review by mpl — APPROVED** [review]
I don't know the github actions, trusting you on that one. :-)

**Pull request merged**